aten/src/ATen/native/quantized/library.cpp

```diff
@@ -61,8 +65,10 @@ TORCH_LIBRARY(_quantized, m) {
   m.def("conv2d(Tensor qx, Tensor weight, int[] stride, int[] padding, int[] dilation, int groups, float output_scale, int output_zero_point) -> Tensor");
   m.def("conv2d_relu(Tensor qx, Tensor weight, int[] stride, int[] padding, int[] dilation, int groups, float output_scale, int output_zero_point) -> Tensor");
   m.def("conv2d_prepack(Tensor weight, Tensor? bias, int[] stride, int[] padding, int[] dilation, int groups) -> Tensor");
-  m.def("linear(Tensor X, Tensor W_prepack, float Y_scale_i, int Y_zero_point_i) -> Tensor Y");
-  m.def("linear_dynamic(Tensor X, Tensor W_prepack) -> Tensor Y");
-  m.def("linear_prepack(Tensor W, Tensor? B=None) -> Tensor W_prepack");
-  m.def("linear_prepack_fp16(Tensor W, Tensor? B=None) -> Tensor W_prepack");
+  m.def("linear(Tensor X, __torch__.torch.classes.quantized.LinearPackedParamsBase W_prepack, float Y_scale_i, int Y_zero_point_i) -> Tensor Y");
```

Comment:
I'm surprised this works, because IIRC all types mentioned in schemas must be defined before you try to register the schema. Maybe a latent SIOF failure here. Better to define the class directly in this `TORCH_LIBRARY` block